### PR TITLE
Travis doc test error (testing)

### PR DESCRIPTION
**Pull request by brechmos-stsci**
There is an issue in the doc test, that is just to see if an empty commit causes the same fail

### CODE REVIEW

**Comment by eteq** [comment]
Note: am trying to figure out what's going on with the doctest failure - it's a known thing that numpy is a bit finickey about spacing.  Just trying to acertain what version of numpy the problem is in.

**Comment by eteq** [comment]
But the astropy-dev failure is real - I'm not sure what the origin of it is, but seems to be a change to nddata.  Will need to diagnose and see if this is important for upstream astropy to know about...

**Comment by brechmos-stsci** [comment]
So @eteq, allow failure for astropy-dev but what about numpy 1.14 do tests against it (as opposed to 1.11)?

I am fine with making astropy-dev accept fail. I am also fine to test against numpy 1.14 based on our discussion yesterday and your comment that we **test** against numpy 1.14 but it should work against older versions.

**Comment by eteq** [comment]
@brechmos-stsci - yeah, *for now* (in this PR) lets set 1.14 as the numpy version and astropy-dev to allow-fail.  Will make another issue to resolve the numpy situation (and #380 already exists for astropy-dev.

**Comment by brechmos-stsci** [comment]
Will do now...

File: .travis.yml [hunk 1]
```diff
@@ -95,6 +91,11 @@ matrix:
         - os: linux
           env: MAIN_CMD='pycodestyle specutils --count' SETUP_CMD=''
 
+        # Try development version of Astropy
```
Comment (bsipocz) [inline .travis.yml]: Anything under `allow_failures` has to be included also in the main matrix above. Basically this change has remove the dev testing altogether.
https://github.com/astropy/specutils/pull/418 should address this.

**Pull request merged**